vision_agent/tools/tools.py

```diff
@@ -1559,6 +1663,74 @@ def overlay_heat_map(
     return np.array(combined)
 
 
+def overlay_counting_results(
+    image: np.ndarray, instances: List[Dict[str, Any]]
+) -> np.ndarray:
+    """'overlay_counting_results' is a utility function that displays counting results on
+    an image.
+
+    Parameters:
+        image (np.ndarray): The image to display the bounding boxes on.
+        instances (List[Dict[str, Any]]): A list of dictionaries containing the bounding
+            box information of each instance
+
+    Returns:
+        np.ndarray: The image with the instance_id dislpayed
+
+    Example
+    -------
+        >>> image_with_bboxes = overlay_counting_results(
+            image, [{'score': 0.99, 'label': 'object', 'bbox': [0.1, 0.11, 0.35, 0.4]}],
+        )
+    """
+    pil_image = Image.fromarray(image.astype(np.uint8)).convert("RGB")
+    color = (158, 218, 229)
+
+    width, height = pil_image.size
+    fontsize = max(10, int(min(width, height) / 80))
```

Comment:
I'd make the font size a little bigger, from the bird image it's difficult to see the count

Reply:
Actually thats an outdated image. I'm attaching new images. I tried different font sizes and when i make it any bigger, its starts to mask the whole object when the objects that are being counted small. This is a normal use case of counting where there are too many instances (crowd, retail objects, fruits etc) and object size gets smaller. If its small, at least there is an option to zoom in and verify but once it masks the object it becomes hard to count overlapping objects
![bird](https://github.com/user-attachments/assets/3c2cbbf5-e9cb-4d2a-9589-aeb532f1ab57)
![count_viz](https://github.com/user-attachments/assets/e1a6547d-0694-4d56-b1b6-f9e006df2d78)